example/ios/Runner/AppDelegate.swift

```diff
@@ -1,20 +1,13 @@
-import UIKit
 import Flutter
-import background_location_tracker
+import UIKit
 
-@UIApplicationMain
+@main
 @objc class AppDelegate: FlutterAppDelegate {
-    override func application(_ application: UIApplication,didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        GeneratedPluginRegistrant.register(with: self)
-
-        if #available(iOS 10.0, *) {
-          UNUserNotificationCenter.current().delegate = self
-        }
-
-        BackgroundLocationTrackerPlugin.setPluginRegistrantCallback { registry in
```

Comment:
Isn't this needed for the example project?

Reply:
This has been re-added